### ISSUE TITLE
Count the number of Divvy stations near a given point

### ISSUE THREAD

**Issue by stevevance** [zorostang/chicago_bike_equity#17]
Install the context-menu plugin for Leaflet to allow the user to right-click a point on the map. Then find the 5 nearest Divvy stations and report their distances.

**Comment by stevevance** [comment]
The Turf.js "nearest" function only finds 1 feature and isn't extensible to find more than one. You can see this live on the map now, and it's also [explained in my blog post](http://www.stevencanplan.com/2015/03/use-turf-to-perform-gis-functions-in-a-web-browser/). 

@zorostang You should pull the latest from master because @rguico expanded your work on the Divvy station buffers to merge them into one polygon. I then added a right-click function that finds the nearest Divvy station to any part on the map.

**Comment by rguico** [comment]
This issue remains open so that we can consider implementing https://github.com/mapbox/leaflet-knn instead.

**Comment by stevevance** [comment]
@rguico Are you working on this?

**Comment by rguico** [comment]
Yes, but heads up: it might be a slower algorithm. If it's decidedly slower I'll let you know.

**Comment by stevevance** [comment]
@rguico Keep in mind there's just a small chance I might want to add bike directions between the point and the Divvy station. The reason I would keep it out is because this isn't a Divvy app. We need to determine the utility of such a function towards our mission before including it.

**Comment by stevevance** [comment]
@rguico Can you update this function to only show those Divvy stations within 1/4 mile? And can you count up to 20 stations to find those within 1/4 mile?

**Comment by rguico** [comment]
I'm on it. I won't be available Tuesday nights for the next 2 weeks because
my parents / Tuesday babysitters are out of town. But I will try to find
time for it.

**Comment by stevevance** [comment]
@rguico Okay, I'll keep that in mind. I'm spending today closing out old issues and if you sync you'll see I integrated the "find nearest bike lane" to the right-click "Get Access Index".

**Comment by stevevance** [comment]
@rguico Can you also make these changes:
- Remove the lines between the source point and the Divvy stations
- Replace the icon with a Divvy icon (attached) 
- Move the line's distance label into the Divvy station popup (I added the popup today)

![icon_divvy](https://cloud.githubusercontent.com/assets/1330459/7597881/25e61b42-f8bc-11e4-90dd-fa33ea2414f6.png)
